pkg/service/webhook/service.go

```diff
@@ -205,10 +207,15 @@ func (s Service) PublishWebhook(noun Noun, verb Verb, payloadReader io.Reader) {
 			continue
 		}
 
-		if err = s.post(ctx, url, string(postJSONData)); err != nil {
-			logrus.WithError(err).Errorf("posting payload to %s", url)
-		}
+		wg.Add(1)
+		go func(url, data string) {
+			defer wg.Done()
+			if err := s.post(ctx, url, data); err != nil {
```

Comment:
Very cool.

If it ever becomes an issue and we have 1000 urls to post to eventually we can do batching / retry logic. For now this is great though